domain/secretbackend/service/service_test.go

```diff
@@ -1005,6 +1006,86 @@ func (s *serviceSuite) TestWatchSecretBackendRotationChanges(c *gc.C) {
 	wC.AssertNoChange()
 }
 
+func (s *serviceSuite) TestGetModelSecretBackend(c *gc.C) {
```

Comment:
There is no tests to cover off on the well known errors that could occur out of this service. We need to assert the contract that we say the GetModelSecretBackend says it implements.

Reply:
There is no point to test those errors because they are not raised from the service itself.